databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksConfig.java

```diff
@@ -581,12 +582,19 @@ private OpenIDConnectEndpoints fetchDefaultOidcEndpoints() throws IOException {
       return new OpenIDConnectEndpoints(prefix + "/v1/token", prefix + "/v1/authorize");
     }
 
-    String oidcEndpoint = getHost() + "/oidc/.well-known/oauth-authorization-server";
-    Response resp = getHttpClient().execute(new Request("GET", oidcEndpoint));
-    if (resp.getStatusCode() != 200) {
-      return null;
-    }
-    return new ObjectMapper().readValue(resp.getBody(), OpenIDConnectEndpoints.class);
+    ApiClient apiClient =
+        new ApiClient.Builder()
+            .withDatabricksConfig(this)
+            .withTimer(new SystemTimer())
+            .withAuthenticateFunc(v -> new HashMap<String, String>())
+            .withGetHostFunc(v -> getHost())
+            .withGetAuthTypeFunc(v -> "")
```

Comment:
This we should probably leave out? Otherwise it is not null according to your check.